packages/next-server/lib/runtime-config.ts

```diff
@@ -4,6 +4,11 @@ export default () => {
   return runtimeConfig
 }
 
-export function setConfig(configValue: any) {
-  runtimeConfig = configValue
+export function setConfig(
+  { publicRuntimeConfig, serverRuntimeConfig } = {} as any
+) {
+  runtimeConfig = {
+    publicRuntimeConfig: publicRuntimeConfig || {},
```

Comment:
When would you need to fall back to `{}` if there is literally no case where it's not set?

Reply:
Per https://github.com/zeit/next.js/pull/7847#discussion_r301797505 `publicRuntimeConfig` isn't present when loading from `__NEXT_DATA__` if it's empty.

Reply:
But it doesn't have to be set either in that case

Reply:
Updated to prevent the bundle size increase